lib/octokit/client/commit_comments.rb

```diff
@@ -32,7 +32,7 @@ def commit_comments(repo, sha, options = {})
       # @return [Sawyer::Resource] Commit comment
       # @see https://developer.github.com/v3/repos/comments/#get-a-single-commit-comment
       def commit_comment(repo, id, options = {})
-        get "#{Repository.path repo}/comments/#{id}", options
+        paginate "#{Repository.path repo}/comments/#{id}", options
```

Comment:
I don't think we need a change here since it's fetching a single commit comment.